deliverables/milestone-4/nathan/tools/Inserter.java

```diff
@@ -72,13 +72,29 @@ public static void main(String[] args) throws SQLException {
 		System.out.format("Reading user: u:%s \n",
 				u1.getUserName());
 		
-		// Update
+		// Update.
 		Users u3 = usersDao.updateUserName(u1, "u2");
 		System.out.format("Updating users: name:%s \n",
 				u3.getUserName());
 		
 		// Delete.
 		// usersDao.delete(p3);
-		
+
+		// INSERT objects from our model.
+		Brewers brewer1 = new Brewers(1234);
+		brewer1 = brewersDao.create(brewer1);
+		Brewers brewer2 = new Brewers(5678);
+		brewer2 = brewersDao.create(brewer2);
+
+		// READ.
+		Brewers b1 = brewersDao.getBrewerById(1234);
+		Assert.assertEquals(b1.getBrewerId(), 1234);
+
+		// Update.
+		Brewers b3 = brewersDao.updateBrewerId(b1, 5678);
+		Assert.assertEquals(b3.getBrewerId(), 5678);
+
+		// Delete.
+		// brewersDao.delete(b3);
```

Comment:
Don't forget to delete the brewers you created so we can run this over and over.

Reply:
have fixed that in the new commit, and delete all four brewers: ```brewer1```, ```brewer2```, ```b1```, and ```b3```